config/database.yml

```diff
@@ -1,25 +1,55 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
+# PostgreSQL. Versions 8.2 and up are supported.
```

Comment:
Reference take from [here](https://gist.github.com/jwo/4512764) and [here](https://medium.com/geekculture/ruby-on-rails-switch-from-sqlite3-to-postgres-590009645c25)

Reply:
use `rake db:create:all` instead of `rake db:create` or `rake db:setup`

1. `rake db:create all` create all databases in all environments (dev, test, prod) mentioned in `database.yml`
2. `rake db:create` only creates database in a particular environment, since we are in dev, so it will only create database in dev environment
3. `rake db:setup` only creates databases in dev and test environments despite mentioning prod in `database.yml`

--- 
Question
1. Why `rake db:setup` only does not create database in prod?

Reply:
Before answering your questions, let me ask you something.

What is prod environment?

Reply:
A production environment is the final product where your end users of the system will be using.

Reply:
Ok. So, will a production environment be running on your machine?

Reply:
No😅

Reply:
haha. does that answer your question? if yes, you can resolve this